tests/install.sh

```diff
@@ -18,15 +18,26 @@ mkdir -p $BUILD_DIR
 
 # Init composer
 composer init --working-dir $BUILD_DIR --no-interaction
+
 composer req --working-dir $BUILD_DIR php-http/discovery --no-update
 
-# Define packages from arguments
-composer req --working-dir $BUILD_DIR $3
+# Argument 3 installs additional composer packages
+if ! [ -z "$3" ]; then
+    composer req --working-dir $BUILD_DIR $3
+else
+    # composer req --no-update impiles --no install
+    composer req --working-dir $BUILD_DIR
```

Comment:
isn't this the same as above when `$3` is `""`? can we simply drop the if/else and revert to the previous code?

Reply:
haha, that's good, i just focus on how to fix that error.

Reply:
that is why we do reviews ;-) 
getting it to run was the hard work!

Reply:
looks good